### ISSUE TITLE
Fix penalty counts / count colors in tcdg16 whiteboard

### ISSUE THREAD

**Issue by Ratzmandious** [rollerderby/scoreboard#263]
Penalty counts and colors for 5/6/7 penalties are not working in the tcdg16 whiteboard.

I'm adding this as the issues are purely JS and CSS based and I have fixed them locally and want an issue to tie them to.

**Comment by JeneralPain** [comment]
Can you share the fix and I’ll do a pull to rectify it?

Thanks

Jeneral Pain

**Comment by frank-weinberg** [comment]
This has been fixed in #208, which should be in the upcoming bugix release.

**Comment by Ratzmandious** [comment]
Looking at the code change for #208, I don't believe it will resolve the penalty count issue in tcdg which was due to the team divs being renamed to Teamd1 and Teamd2 under Teamd, and the totalbox variable being called "totalboxd".

The colors for 5/6/7 penalties were commented out (probably because they were too vibrant as they set the background color rather than the text color). I reinstated them setting the text color instead.

[tcdg2016wb.zip](https://github.com/rollerderby/scoreboard/files/3174012/tcdg2016wb.zip)
